Emby.AutoOrganize/Data/ManagedConnection.cs

```diff
@@ -7,15 +7,22 @@
 
 namespace Emby.AutoOrganize.Data
 {
-    public class ManagedConnection :  IDisposable
+    /// <summary>
+    /// Wraps a <see cref="SQLiteDatabaseConnection"/>.
+    /// </summary>
+    public sealed class ManagedConnection : IDisposable
```

Comment:
These classes have been copied from https://github.com/jellyfin/jellyfin/tree/master/Emby.Server.Implementations/Data maybe you could reuse this documentation there ;)